.wordlist.txt

```diff
@@ -0,0 +1,187 @@
+APIs
+AST
+AdministratorAccess
+Avro
+CIDR
+DDL
+DSL
+DSLs
+DataFrame
+DataFrames
+Datadog
+Dockerfile
+Flink
+GCP
+GCP's
+GRPC
+Github
+Grafana
+Graviton
+Groupby
+IOPS
+InfoSec
+Instacart
+JSON
+JSX
+JVM
+Kaggle
+Kubernetes
+LHS
+LastK
+MockClient
+Nones
+OAuth
+OOM
+OpenSSL's
+PII
+PLAINTEXT
+PagerDuty
+PoolableConnectionFactory
+PrivateLink
+Pulumi
+PyO
+Pydantic
+RHS
+ROI
+RPCs
+Realtimeliness
+RocksDB
+SASL
+SDK
+SHA
+SLA
+SSL
+SearchRequest
+Signifier
+Stddev
+TLS
+TLSv
+TestCase
+TestDataset
+Tokio
+Tokio's
+UI
+UserCreator
+UserCreditScore
+UserFeature
+UserFeatures
+UserInfo
+UserInfoDataset
+UserLocation
+UserPost
+UserTransactionsAbroad
+VPC
+WIP
+WIP
+YAML
+ai
+api
+architected
+assertEqual
+async
+autoscaling
+backend
+backfill
+backfilled
+bmi
+bmr
+bool
+boolean
+booleans
+classmethod
+classmethods
+codebase
+codepaths
+compilable
+config
+configs
+csv
+dataclass
+dataflow
+dataframe
+dataset
+dataset's
+datasets
+datastore
+datastores
+datetime
+dateutil
+declaratively
+dedup
+denormalize
+dev
+df
+dfe
+docsnip
+ds
+durations
+embeddings
+enabledTLSProtocols
+featureset
+featuresets
+fintech
+frontend
+geocoding
+geoid
+groupby
+gserviceaccount
+hackathon
+hardcoded
+html
+hudi
+iam
+ip
+ish
+ith
+jdbc
+json
+kafka
+kwarg
+kwargs
+latencies
+lifecycle
+lookup
+lookups
+metaflags
+multicolumn
+mysql
+nan
+natively
+noqa
+np
+nullable
+params
+parseable
+pid
+postgres
+pre
+precompute
+precomputed
+protobuf
+protobufs
+quickstart
+realtime
+regex
+regexes
+repo
+runtime
+scalability
+scalable
```

Comment:
Can we add extractor, depends_on, version, inputs, ( all our sources maybe ? kinesis, bigquery etc )

Reply:
Most of those are already part of the dictionary -- I'm not quite sure why `scalable` and `scalability` aren't. It's just this text file (which doesn't even need to be sorted) so trivial to add things later.